nodes/src/nodes/llm_vision_anthropic/anthropic_vision.py

```
@@ -0,0 +1,235 @@
+        """Initialize the Anthropic Vision chat instance."""
+        super().__init__(provider, connConfig, bag)
+        config = Config.getNodeConfig(provider, connConfig)
+
+        self._model = config.get('model', 'claude-opus-4-6')
+        api_key = config.get('apikey')
+
+        self._system_prompt = config.get('vision.systemPrompt') or config.get('systemPrompt') or ''
+        self._prompt = config.get('vision.prompt') or config.get('prompt') or 'Describe this image in detail.'
+
+        if not api_key:
+            raise ValueError('Missing Anthropic API key. Get one at https://console.anthropic.com')
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)